src/github.com/getlantern/flashlight/genconfig/checkfallbacks.go

```diff
@@ -0,0 +1,166 @@
+// Utility for checking if fallback servers are working properly.
+// It outputs failing servers info in STDOUT.  This allows this program to be
+// used for automated testing of the fallback servers as a cron job.
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"runtime"
+	"strings"
+	"sync"
+
+	"github.com/getlantern/golog"
+
+	"github.com/getlantern/flashlight/client"
+)
+
+var (
+	help          = flag.Bool("help", false, "Get usage help")
+	fallbacksFile = flag.String("fallbacks", "fallbacks.json", "File containing json array of fallback information")
+	verbose       = flag.Bool("verbose", false, "Verbose output for debugging")
+	numConns      = flag.Int("connections", 1, "Number of simultaneous connections")
+
+	expectedBody = "Google is built by a large team of engineers, designers, researchers, robots, and others in many different sites across the globe. It is updated continuously, and built with more tools and technologies than we can shake a stick at. If you'd like to help us out, see google.com/careers.\n"
+)
+
+var (
+	log = golog.LoggerFor("checkfallbacks")
+)
+
+type fallbackServer map[string]interface{}
+
+func main() {
+	flag.Parse()
+
+	if *help {
+		flag.Usage()
+		os.Exit(1)
+	}
+
+	numcores := runtime.NumCPU()
+
+	if *verbose {
```

Comment:
Nice, although if log debug goes to stderr maybe we don't need to care about limiting log output.